easybuild/easyconfigs/h/HEALPix/HEALPix-3.50-foss-2018b.eb

```diff
@@ -0,0 +1,54 @@
+name = 'HEALPix'
+version = '3.50'
+
+homepage = 'http://healpix.sourceforge.net/'
+description = """Hierarchical Equal Area isoLatitude Pixelation of a sphere."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['Healpix_3.50_2018Dec10.tar.gz']
+patches = ['HEALPix_noXtest.patch']
+checksums = [
+    'ec9378888ef8365f9a83fa82e3ef3b4e411ed6a63aca33b74a6917c05334bf4f',  # Healpix_3.50_2018Dec10.tar.gz
+    'ec82c8b2beb80937c83d2e545e26ade08b3647ab5a3401c8703e6523b0782c08',  # HEALPix_noXtest.patch
+]
+
+dependencies = [('CFITSIO', '3.45')]
+
+sanity_check_paths = {
+    'files': [
+        'bin/alteralm',
+        'bin/anafast',
+        'bin/hotspot',
+        'bin/map2gif',
+        'bin/median_filter',
+        'bin/plmgen',
+        'bin/sky_ng_sim',
+        'bin/sky_ng_sim_bin',
+        'bin/smoothing',
+        'bin/synfast',
+        'bin/ud_grade',
+        'lib/libchealpix.a',
+        'lib/libgif.a',
+        'lib/libhealpix.a',
+        'lib/libhpxgif.a',
+        'lib/libchealpix.so',
```

Comment:
So the only difference here with the sanity check that is now done only for older versions (cfr. https://github.com/easybuilders/easybuild-easyblocks/pull/1752) is not checking for `lib/libpsht_healpix_f.a`?

This could be handled better in the easyblock imho, which is do once and forget about it (until they change more stuff in the future, I guess).

Even simply dropping the check for `lib/libpsht_healpix_f.a` without any version check makes more sense to me than moving these paths to the easyconfig file, which will bite us later w.r.t. maintenance...

Reply:
> dropping the check for `lib/libpsht_healpix_f.a` without any version check makes more sense 
> ..
> until they change more stuff in the future, I guess

ok, here I was more like "don't break existing stuff", and tried to keep the easyblock as much general as it could be.. In the current configure script they introduced "experimental feature" - producing dynamic `*.so` libs instead(!) static `*.a`, so I somehow expect they're going to change the file list in next version again.. 

`<opinion>`

Generally speaking - I think in case of software with good code and build culture I agree with you, and it's good to do as much as possible in the easyblock. But not fully sure if this is the case of Healpix. 

I believe one should be able to jump between versions just with modification of easyconfig file.

`</opinion>`

Reply:
solved in https://github.com/easybuilders/easybuild-easyblocks/pull/1763